circle.yml

```diff
@@ -9,8 +9,14 @@ deployment:
   release:
     tag: /v[0-9]+(\.[0-9]+)*/
     commands:
-        - ./publish release $CIRCLE_TAG
+        - yarn build:
+            environment:
+              REACT_APP_API_URL: api.codesplain.io/prod
```

Comment:
I don't think we want the prod API to be a `/prod` path.  I would say `/api` or i can look into having no path route to the prod API